otj-kafka/src/main/java/com/opentable/kafka/logging/LoggingUtils.java

```diff
@@ -71,6 +71,8 @@
     private final String javaVersion;
     private final String os;
     private final Bucket errLogging;
+    // See comments under addHeader - TODO: integrate with Opentracing context
+    private final String traceId = UUID.randomUUID().toString();
```

Comment:
As noted, this is simply set on instantiation. It would eventually hook into Opentracing's context